examples/evkbmimxrt1060/test/app_main.c

```diff
@@ -70,6 +68,25 @@ extern void vCLITask( void * pvParam );
 
 extern void vOTAUpdateTask( void * pvParam );
 
+int RunDeviceAdvisorDemo( void )
+{
+    BaseType_t xResult = pdFAIL;
+    if( xResult == pdPASS )
+    {
+        xResult = xMQTTAgentInit( appmainMQTT_AGENT_TASK_STACK_SIZE, appmainMQTT_AGENT_TASK_PRIORITY );

```

Comment:
MQTT Agent is required for running OTA E2E tests as well. So you may want to call this outside of this function.